package.json

```diff
@@ -15,44 +15,52 @@
   "scripts": {
     "start": "node node_modules/grunt-cli/bin/grunt",
     "test": "node node_modules/grunt-cli/bin/grunt test",
-    "postinstall": "node node_modules/grunt-cli/bin/grunt install"
+    "postinstall": "node node_modules/grunt-cli/bin/grunt install",
+    "karma:test": "karma start test/karma.conf.js"
```

Comment:
Can we use a more user-friendly and easy to understand _npm-script-command_ I would suggest something in this manner _test:client_ or _test:frontend_ This way we can anticipate an expectation when it is used.